lib/now.ts

```diff
@@ -1,40 +1,40 @@
-import { ES } from './ecmascript.mjs';
-import { GetIntrinsic } from './intrinsicclass.mjs';
+import { ES } from './ecmascript';
+import { GetIntrinsic } from './intrinsicclass';
 
 const instant = () => {
   const Instant = GetIntrinsic('%Temporal.Instant%');
   return new Instant(ES.SystemUTCEpochNanoSeconds());
 };
-const plainDateTime = (calendarLike, temporalTimeZoneLike = timeZone()) => {
+const plainDateTime = (calendarLike, temporalTimeZoneLike = defaultTimeZone()) => {
```

Comment:
Out of curiosity, why was this renaming needed?

Reply:
```
Parameter 'temporalTimeZoneLike' cannot reference identifier 'timeZone' declared after it.ts(2373)
Block-scoped variable 'timeZone' used before its declaration.ts(2448)
now.ts(9, 9): 'timeZone' is declared here.
```

It seems that tsc manages to confuse the timeZone function and the constant declared within the function body, so I renamed the function.

Reply:
In this case, I'd suggest renaming the local variables `tz` instead of renaming the `timeZone` function. This would be less change vs. JS, and also IMHO it'd be more clear that the default params are calling the implementation of `Now.timeZone` as opposed to some other function.

Reply:
Done - renamed as you suggested.